17.json

```diff
@@ -36,12 +36,12 @@
 
   "errorNoPayloads": "Nenhum payload disponível :(",
 
-  "errorSendPayloadOpen": "",
-  "errorSendPayloadBuffer": "",
-  "errorSendPayloadRead": "",
-  "errorSendPayloadCreateSocket": "",
-  "errorSendPayloadConnect": "",
-  "errorSendPayloadSend": "",
+  "errorSendPayloadOpen": "Falha ao abrir o arquivo da payload",
+  "errorSendPayloadBuffer": "Falha ao alocar a memória para o buffer da payload",
+  "errorSendPayloadRead": "Falha na leitura ao copiar o payload para o buffer",
```

Comment:
I'm supposing here that the Payload Guest are copying the payload to a connection buffer